mlflow/langchain/langchain_tracer.py

```diff
@@ -203,8 +196,6 @@ def on_llm_start(
         """Run when LLM (non-chat models) starts running."""
         if metadata:
             kwargs.update({"metadata": metadata})
-        if _USE_MLFLOW_LANGCHAIN_TRACER_FOR_RAG_TRACING.get():
```

Comment:
@sunishsheth2009 I dropped all these conversions. get_databricks_vector_search_key is not used anywhere in mlflow then, do you still need them or we can clean it up?